src/figuresClasses.ts

```diff
@@ -1,11 +1,77 @@
-export interface Figure {}
+export interface Figure {
+  color: string;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  public shape: string = 'triangle';
+
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    if (a <= 0 || b <= 0 || c <= 0) {
+      throw new Error('your error message');
```

Comment:
[CODE STYLE] - When throwing errors, provide a detailed description in the error message to help understand the exact nature of the error. For example: 'Triangle side lengths must be greater than 0.'